src/admin.js

```diff
@@ -33,6 +31,22 @@ export class StitchAdminClient extends StitchClient {
     return ADMIN_CLIENT_TYPE;
   }
 
+  get _v1() {
+    const v1do = (url, method, options) =>
+      super._do(url, method, Object.assign({}, { apiVersion: v1, apiType: API_TYPE_PRIVATE }, options)).then(response => {
```

Comment:
[nit] instead of `Object.assign` here you could just do `{ apiVersion: v1, apiType: API_TYPE_PRIVATE, ...options}`